homeworks/03-data-structure/tests/BinarySearchTree.test.ts

```diff
@@ -0,0 +1,66 @@
+import { BinarySearchTree } from '../src/BinarySearchTree';
+
+describe('BinarySearchTree', () => {
+  describe('insertValue', () => {
+    it('Create a Search Tree', () => {
+      const tree = new BinarySearchTree(8);
+      tree.adjoin(3, 10, 1, 6, 14, 4, 7, 13);
+
+      expect(tree).toEqual({
+        value: 8,
+        left: {
+          value: 3,
+          left: { value: 1 },
+          right: {
+            value: 6,
+            left: { value: 4 },
+            right: { value: 7 },
+          },
+        },
+        right: {
+          value: 10,
+          right: {
+            value: 14,
+            left: { value: 13 },
+          },
+        },
+      });
+    });
+  });
+  describe('acquire', () => {
```

Comment:
It is a good practice to insert empty line between blocks of code to make it more readable if the code exceeds at least half a screen.